src/components/app.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import url from 'url';
+
+import '../../node_modules/jquery/src/jquery';
 import '../../node_modules/uswds/dist/js/uswds.js';
+import '../../node_modules/bootstrap/dist/js/bootstrap.js';
+
 import '../../node_modules/@cmsgov/design-system-core/dist/index.css';
+import '../../node_modules/bootstrap/dist/css/bootstrap.css';
 
+import '../styles/qpp-style/css/qpp-style.css';
```

Comment:
could we import these in the header and footer files, or does app.jsx use qpp styles?